knowledge/fast_food/hungry_jacks/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: restaurant
+seed_examples:
+- answer: Franchise of Burger King and owned by Competitive Food Australia|
+  question: Who owns Hungry Jack's Pty?
+- answer: Australia|
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!